package.json

```diff
@@ -21,7 +21,9 @@
     "test:int": "jest -c ./test/jest.integration.config.ts",
     "test:e2e": "jest -c ./test/jest.e2e.config.ts",
     "test:sec": "jest -c ./test/jest.security.config.ts",
-    "test:mig": "jest -c ./test/jest.migration.config.ts"
+    "test:mig": "jest -c ./test/jest.migration.config.ts",
+    "cypress:open": "cypress open",
+    "cypress:run": "cross-env HEIGHT=1080 WIDTH=1920 cypress run"
```

Comment:
What are those for? Can we have a yarn test:webchat

Reply:
This was recommended by the cypress doc, but yes we can. What do you suggest?

Reply:
I suggest we have a one liner way of starting the server, starting the webchat and running the tests. A script could do the job or anything else. The process of running the e2e in botpress is way too complicated and it would be better to have it streamlined here

Reply:
yarn test:chat

It would be awesome if it could display the tests it's doing automatically aswell